task/gitlab-set-status/0.1/gitlab-set-status.yaml

```diff
@@ -0,0 +1,140 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: gitlab-set-status
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Messaging
```

Comment:
shouldn't the category be `Git` as other tasks of this type are falling under `Git` category

Reply:
I saw it more as sending a message like we do with Slack https://github.com/tektoncd/catalog/blob/af28e02840fde30b0a1f9ccfb37590e8ad0a3e77/task/send-to-channel-slack/0.1/send-to-channel-slack.yaml this time it just happens to be to Gitlab. However, I now see `github-set-status` has `Git` as a category, and I agree it makes the most sense to use the same category for either of them. I'll update it.